src/storage/src/hummock/file_cache/store.rs

```diff
@@ -23,9 +23,9 @@ use foyer::common::code::{Key, Value};
 use foyer::storage::admission::rated_random::RatedRandomAdmissionPolicy;
 use foyer::storage::admission::AdmissionPolicy;
 use foyer::storage::event::EventListener;
-use foyer::storage::store::{FetchValueFuture, PrometheusConfig};
+pub use foyer::storage::metrics::set_metrics_registry;
```

Comment:
May alias it as `set_foyer_metrics_registry`. It is confusing to call a method from the storage crate with name `set_metrics_registry`.